### DISCUSSION TITLE
Is there a way to have 2 databases side by side with the same username and password?

### DISCUSSION THREAD

**Discussion by thenewdesign** [planetscale/discussion#373] [Q&A] [answered]
I have a unique requirement to house two databases together and do performant cross-db joins. Is this supported?

**Comment by fooforge** [comment, marked as answer]
Hey @thenewdesign,

That's not yet supported, I'm afraid. Every PlanetScale database is its own Vitess cluster and runs exactly one database. We have seen requests for these scenarios before and we're looking into ways to support them, but I cannot say if or when we're going to get to it at this time.
